specification/advisor/resource-manager/Microsoft.Advisor/preview/2016-07-12-preview/advisor.json

```diff
@@ -398,6 +398,72 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Advisor/advisorScore": {
+      "get": {
+        "tags": [
+          "AdvisorScore"
+        ],
+        "description": "Retrieves data about the advisor score of the subscription",
```

Comment:
Gets the list of advisor score.

Follow the convention similar to other api
https://docs.microsoft.com/en-us/rest/api/advisor/recommendationmetadata/list